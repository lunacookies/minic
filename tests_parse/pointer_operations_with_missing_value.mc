func main {
	address_of := &
	return a
	dereference := *
	return a
}
