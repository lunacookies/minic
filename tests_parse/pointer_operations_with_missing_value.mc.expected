func main {
	address_of := &(<missing>)
	return a
	dereference := *(<missing>)
	return a
}
tests_parse:28..29: error: missing expression
tests_parse:56..57: error: missing expression
